debian/control

```diff
@@ -35,3 +35,30 @@ Depends: nntrainer, ${shlibs:Depends}, ${misc:Depends}
 Description: NNTrainer Applications pacakge
  This is nntrainer applications package.
 
+Package: nntrainer-ccapi
```

Comment:
need to change the name? ccapi-ml-training?

Reply:
updated to match with tizen packaging - ccapi-ml-training.